framework/resources/Functional/query_parser/DRILL-6974_set_returns_current_value_of_property.sql

```diff
@@ -0,0 +1,3 @@
+set `store.format`='json';
+alter system set `store.format`;
```

Comment:
This could be risky. System option changes can impact other tests. Here CTAS tests may be forced to write to JSON, depending on when this test is executed.

Reply:
At current test we do not change system properties. We change session properties and verify that system property does not change. Also, at the third line we return session property to its default value.